pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```diff
@@ -176,3 +181,6 @@ spec:
         - name: host-run-mount
           hostPath:
             path: /run/mount
+        - name: etc-selinux
+          hostPath:
+            path: /etc/selinux
```

Comment:
will this path be present on all the distro and this path need to be configurable?

Reply:
I didnt get the question @Madhu-1 , can you please elaborate?

Reply:
Is the `/etc/selinux` path present on all the Linux distributions? if not deployment will fail?

Reply:
if selinux available system , it will be present. thus guaranteed to be working in platforms like openshift. For kube, we are defaulting to `false` and user can enable, if they are running selinux enabled cluster and if they want . does that answer @Madhu-1 ?

Reply:
my question is is it always `/etc/selinux`? i see from some link its at `/etc/sysconfig/selinux` if it's not constant this need to be a configurable parameter for the user

Reply:
@Madhu-1 if you can share I can check the link, but to my knowledge its a symlink and we dont have to worry about it.

Reply:
here are the links I found about it https://www.strongdm.com/docs/architecture/deployment/selinux https://www.omnisecu.com/gnu-linux/redhat-certified-engineer-rhce/security-enhanced-linux-selinux-configuration-file.php

Reply:
as in the link https://www.omnisecu.com/gnu-linux/redhat-certified-engineer-rhce/security-enhanced-linux-selinux-configuration-file.php ```The SELinux configuration file is “/etc/sysconfig/selinux”, which is a symbolic link to “/etc/selinux/conf” file.``` , so should be fine.